src/main/java/uk/gov/companieshouse/confirmationstatementapi/client/OracleQueryClient.java

```diff
@@ -21,23 +26,41 @@
 
     public Long getCompanyTradedStatus(String companyNumber) {
         var getCompanyTradedStatusUrl = String.format("%s/company/%s/traded-status", oracleQueryApiUrl, companyNumber);
-        LOGGER.info("Calling Oracle Query API URL (get): {}", getCompanyTradedStatusUrl);
+        LOGGER.info(CALLING_ORACLE_QUERY_API_URL_GET, getCompanyTradedStatusUrl);
 
         ResponseEntity<Long> response = restTemplate.getForEntity(getCompanyTradedStatusUrl, Long.class);
         var companyTradedStatus = response.getBody();
-        LOGGER.info("Received {} from Oracle Query API URL (get): {}", companyTradedStatus, getCompanyTradedStatusUrl);
+        LOGGER.info(RECEIVED_FROM_ORACLE_QUERY_API_URL_GET, companyTradedStatus, getCompanyTradedStatusUrl);
 
         return companyTradedStatus;
     }
 
     public Integer getShareholderCount(String companyNumber) {
         var shareholderCountUrl = String.format("%s/company/%s/shareholders/count", oracleQueryApiUrl, companyNumber);
-        LOGGER.info("Calling Oracle Query API URL (get): {}", shareholderCountUrl);
+        LOGGER.info(CALLING_ORACLE_QUERY_API_URL_GET, shareholderCountUrl);
 
         ResponseEntity<Integer> response = restTemplate.getForEntity(shareholderCountUrl, Integer.class);
         var count = response.getBody();
-        LOGGER.info("Received {} from Oracle Query API URL (get): {}", count, shareholderCountUrl);
+        LOGGER.info(RECEIVED_FROM_ORACLE_QUERY_API_URL_GET, count, shareholderCountUrl);
 
         return count;
     }
+
+    public StatementOfCapital getStatmentOfCapitalData(String companyNumber) throws ServiceException {
+        var statementOfCapitalUrl = String.format("%s/company/%s/statement-of-capital", oracleQueryApiUrl, companyNumber);
+        LOGGER.info(CALLING_ORACLE_QUERY_API_URL_GET, statementOfCapitalUrl);
+
+        ResponseEntity<StatementOfCapital> response = restTemplate.getForEntity(statementOfCapitalUrl, StatementOfCapital.class);
+        if(response.getStatusCode() == HttpStatus.OK) {
+            StatementOfCapital statementOfCapital = response.getBody();
+            if (statementOfCapital != null) {
+                LOGGER.info(RECEIVED_FROM_ORACLE_QUERY_API_URL_GET, statementOfCapital, statementOfCapitalUrl);
```

Comment:
if you are expecting to get any useful info from putting statement of capital in the logs its going to need to override toString